source/_docs/blueprint/selectors.markdown

```diff
@@ -737,6 +733,142 @@ entity:
+      type: string
+      required: false
+    manufacturer:
+      description: >
+        When set, it limits the list of floor that have devices by the set
+        manufacturer name.
+      type: string
+      required: false
+    model:
+      description: >
+        When set, it limits the list of floors that have devices which have
+        the set model.
+      type: string
+      required: false
+entity:
+  description: >
+    When entity options are provided, the list of floors is filtered by floors
+    that at least have one entity that matches the given conditions. Can be
+    either a object or a list of object.
+  type: list
+  required: false
+  keys:
+    integration:
+      description: >
+        Can be set to an integration domain. Limits the list of floors that
+        have entities by the set integration domain, for example,
+        [`zha`](/integrations/zha).
+      type: string
+      required: false
+    domain:
+      description: >
+        Limits the list of floors that have entities of a certain domain(s),
+        for example, [`light`](/integrations/light) or
+        [`binary_sensor`](/integrations/binary_sensor). Can be either a string
+        with a single domain, or a list of string domains to limit the selection to.
+      type: [string, list]
+      required: false
+    device_class:
+      description: >
+        Limits the list of floors to floors that have entities with a certain
+        device class(es), for example, `motion` or `window`. Can be either a string
+        with a single device_class, or a list of string device_class to limit
+        the selection to.
+      type: [device_class, list]
+      required: false
+    supported_features:
+      description: >
+        Limits the list of floors to floors that have entities with a certain
+        supported feature, for example, `light.LightEntityFeature.TRANSITION`
+        or `climate.ClimateEntityFeature.TARGET_TEMPERATURE`. Should be a list
+        of features.
+      type: list
+      required: false
+multiple:
+  description: >
+    Allows selecting multiple floors. If set to `true`, the resulting value of
+    this selector will be a list instead of a single string value.
+  type: boolean
+  default: false
+  required: false
+{% endconfiguration %}
+
+The output of this selector is the floor ID, or (in case `multiple` is set to
+`true`) a list of floor IDs.
+
+```yaml
+# Example floor selector output result, when multiple is set to false
+first_floor
+
+# Example floor selector output result, when multiple is set to true
+- first_floor
+- second_floor
+```
+
+### Example floor selectors <!-- omit from toc -->
+
+An example floor selector only shows floors that have one or more lights or
+switches provided by the [ZHA](/integrations/zha) integration.
+
+```yaml
+floor:
+  entity:
+    integration: zha
+    domain:
+      - light
+      - switch
+```
+
+Another example uses the are floor selector, which only shows floors that
```

Comment:
```suggestion
Another example uses the are floor selector, which only shows floors that
```
Something went wrong here. Not sure if it's just a typo or if the idea was to refer to a specific floor selector?

Reply:
```suggestion
Another example using the floor selector, which only shows floors that
```